### PR TITLE
Week 6-7 progress update: Mudit Gupta

### PR DESCRIPTION
**Pull request by mg14777**
@wenmogu @craaaa Ready for review.

### CODE REVIEW

**Comment by mg14777** [comment]
This still needs to be merged

File: Mudit_Gupta/progress.md [hunk 1]
```diff
@@ -37,24 +37,43 @@ Week 5 Plan:
...
+
+Week 6 Retrospect:
+Had to spend some time resolving merge conflicts for drag and drop PR due to previous enhancements made to distribute points questions. 
 
 Week 7 Plan:
-- [ ]
-- [ ] Front end development for enhancements to Getting started guide
-- [ ] Writing Selenium tests
+- [x] Writing Selenium tests for drag and drop feature for MSQ and Rank questions
+- [x] Finish work on [Student: Feedback submission: Rank question: Improve explanation regarding ranking constraints](https://github.com/TEAMMATES/teammates/pull/8870)
```
Comment (wkurniawan07) [inline Mudit_Gupta/progress.md]: Do not change the plan.

File: Mudit_Gupta/progress.md [hunk 2]
```diff
@@ -37,24 +37,43 @@ Week 5 Plan:
 - [ ] Fix issues after code review
 
 Week 6 Plan:
-- [ ] Mockups for enhancements to Getting started guide
-- [ ] Develop use cases/user stories + UI workflows for enhancements to Getting started guide
-- [ ] (In depth discussion to ensure common ground on design)
+- [x] Mockups for enhancements to Feedback session creation
+- [x] In depth discussion to ensure common ground on design
+- [x] Implement drag and drop feature for other question types
```
Comment (wkurniawan07) [inline Mudit_Gupta/progress.md]: Do not change the plan.

**Review by wkurniawan07 — CHANGES_REQUESTED** [review]
You should not change the plan. If the week's work does not align with the plan, use the retrospect to reflect on/explain why.

**Comment by mg14777** [comment]
@wkurniawan07 Ready for review. I have kept the original plan and added explanations.

**Comment by mg14777** [comment]
@wkurniawan07 can we merge this ?

**Comment by mg14777** [comment]
@wkurniawan07 @xpdavid @wenmogu Please merge this

**Review by wkurniawan07 — APPROVED** [review]

**Pull request merged**